gpu/build.rs

```diff
@@ -0,0 +1,92 @@
+#[cfg(feature = "cuda")]
+fn compile_cuda_shaders() {
+    use std::process::Command;
+    use walkdir::WalkDir;
+    const CUDA_SOURCE_DIR: &str = "src/cuda/shaders";
```

Comment:
Does the build script always run from the `CARGO_MANIFEST_DIR`? Or could it be possible that running from `src` breaks this? If the latter, we could fix it reading `CARGO_MANIFEST_DIR` first.

Reply:
I don't really know, but I changed it to use `CARGO_MANIFEST_DIR` so it's more robust.